src/timings/Timings.php

```diff
@@ -299,4 +306,52 @@ public static function getEventHandlerTimings(string $event, string $handlerName
 
 		return self::$eventHandlers[$event][$handlerName];
 	}
+
+	public static function getAsyncTasksProgressUpdateTimings(AsyncTask $task, string $group = self::GROUP_BREAKDOWN) : TimingsHandler{
+		if (!isset(self::$schedulerAsync)){
+			self::init();
+		}
```

Comment:
You can do this inside the isset() block below instead of doing it on every call.